std/os/uefi/protocols/simple_text_output_mode.zig

```diff
@@ -0,0 +1,8 @@
+pub const SimpleTextOutputMode = struct {
```

Comment:
Should this be an `extern struct`?

Reply:
All the structs in std/os/uefi probably should be `extern struct`